src/app/app.component.html

```diff
@@ -7,7 +7,7 @@
       [ngClass]="{ 'loading-dots': status === 'connecting' || status === 'initializing' }"
       >{{ status }}</span
     >
-    <span class="splash-screen__hint" *ngIf="showConnectionHint">
+    <span class="splash-screen__hint" *ngIf="showConnectionHint" i18n>
```

Comment:
Can we use custom IDs for these as mentioned here: https://angular.io/guide/i18n#mark-element-attributes-for-translations. This would make adding additional translations tremendously more easy, plus the Angular generated IDs are being overwritten each time `ng extract` is being run.

Reply:
i had started doing it initially but it was so much slower to understand the context to find a good id... i guess i'll do it anyway lol

Reply:
Yeah it is a bit tedious to setup, but it will make future translations and keeping everything up to date so much easier :)